docs/comparison-pgvector.md

```diff
@@ -1,5 +1,15 @@
 # Comparison with pgvector
 
+|                                             | pgvecto.rs                                             | pgvector                |
+| ------------------------------------------- | ------------------------------------------------------ | ----------------------- |
+| Transaction support                         | ✅                                                      | ⚠️                       |
+| Sufficient Result with Delete/Update/Filter | ✅                                                      | ⚠️                       |
+| Vector Dimension Limit                      | 65535                                                  | 2000                    |
+| Prefilter on HNSW                           | ✅                                                      | ❌                       |
+| Parallel Index build                        | ⚡️ Linearly faster with more cores                      | 🐌 Only single core used |
+| Async Index build                           | Ready for queries anytime and do not block insertions. | ❌                       |
+| Quantization                                | Scalar/Product Quantization                            | ❌                       |
+
```

Comment:
Why dont we keep it in README?

Reply:
README is too long before.

Reply:
I think we can keep this table in readme?

Reply:
moved back